baremetal_network_provisioning/db/migration/alembic_migrations/versions/3297cd3f2323_bm_nw_provision_models.py

```diff
@@ -84,31 +72,51 @@ def upgrade():
                     sa.Column('security_name', sa.String(255), nullable=True),
                     sa.Column('auth_protocol', sa.String(16), nullable=True),
                     sa.Column('auth_key', sa.String(255), nullable=True),
                     sa.Column('priv_protocol', sa.String(16), nullable=True),
                     sa.Column('priv_key', sa.String(255), nullable=True),
                     sa.Column('security_level', sa.String(16), nullable=True),
-                    sa.ForeignKeyConstraint(
-                        ['id'],
-                        ['bnpphysicalswitchports.switch_id'],
-                        ondelete='CASCADE'),
                     sa.PrimaryKeyConstraint('id'))
 
-    op.create_table('bnpswitchportmappings',
+    op.create_table('bnp_physical_switch_ports',
+                    sa.Column('id', sa.String(36), nullable=False),
+                    sa.Column('switch_id', sa.String(36), nullable=False),
+                    sa.Column('interface_name', sa.String(255),
+                              nullable=False),
+                    sa.Column('ifindex', sa.String(255), nullable=False),
+                    sa.Column('port_status', sa.String(16), nullable=False),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint('switch_id', 'interface_name'),
+                    sa.ForeignKeyConstraint(
+                        ['switch_id'],
+                        ['bnp_physical_switches.id'],
+                        ondelete='CASCADE'))
+
+    op.create_table('bnp_switch_port_mappings',
                     sa.Column('neutron_port_id', sa.String(36),
                               nullable=False),
                     sa.Column('switch_port_id', sa.String(36),
                               nullable=False),
                     sa.Column('switch_id', sa.String(36), nullable=False),
-                    sa.UniqueConstraint('neutron_port_id', 'switch_port_id'))
+                    sa.UniqueConstraint('neutron_port_id', 'switch_port_id'),
+                    sa.ForeignKeyConstraint(
+                        ['switch_port_id'],
+                        ['bnp_physical_switch_ports.id'],
+                        ondelete='CASCADE'))
 
-    op.create_table('bnpneutronports',
+    op.create_table('bnp_neutron_ports',
                     sa.Column('neutron_port_id', sa.String(36),
                               nullable=False),
                     sa.Column('lag_id', sa.String(36), nullable=True),
                     sa.Column('access_type', sa.String(16), nullable=False),
                     sa.Column('segmentation_id', sa.Integer, nullable=False),
                     sa.Column('bind_status', sa.Boolean(), nullable=True),
-                    sa.PrimaryKeyConstraint('neutron_port_id'))
+                    sa.PrimaryKeyConstraint('neutron_port_id'),
+                    sa.ForeignKeyConstraint(
+                        ['neutron_port_id'],
```

Comment:
This table should have neutron_port_id as the primary key.  The other table bnp_switch_port_mappings should refer to this.

The statement here looks otherwise.